lms/djangoapps/instructor_task/api.py

```diff
@@ -337,7 +337,7 @@ def submit_calculate_students_features_csv(request, course_key, features):
     """
     task_type = 'profile_info_csv'
     task_class = calculate_students_features_csv
-    task_input = {'features': features}
+    task_input = features
```

Comment:
Given the urgency of this fix, I'm willing to apply the band-aid for now. However, I think we need to prioritize the longer term fix — we recently merged an open source PR that added things to the task input, and I think in general we should get better at not bloating it. Especially so that when people write new tasks, they'll have good models to base them off of.